x-pack/plugins/cases/server/client/cases/validators.ts

```diff
@@ -95,12 +95,12 @@ export const validateRequiredCustomFields = ({
   requestCustomFields,
   customFieldsConfiguration,
 }: CustomFieldValidationParams) => {
-  if (!Array.isArray(requestCustomFields) || !requestCustomFields.length) {
-    return;
-  }
-
   if (customFieldsConfiguration === undefined) {
-    throw Boom.badRequest('No custom fields configured.');
+    if (!Array.isArray(requestCustomFields) || !requestCustomFields.length) {
```

Comment:
I am a bit confused about this change. What scenario are we trying to cover?

Reply:
This is strange, this doesn't seem to be my change 😄

Reply:
It seems that @adcoelho try to fix some tests in [`f10dd1b` (#167472)](https://github.com/elastic/kibana/pull/167472/commits/f10dd1b09d12f1c3aa3c7101405ff2ad513e209e)

Reply:
I think that if there is no configuration or no custom fields we should return. The function validates only required custom fields so no configuration means no custom fields are required.

Reply:
We discussed offline and we decided to leave it as it is.